docs/source/array-creation.rst

```diff
@@ -352,6 +352,45 @@ Files may be retrieved by running `da.from_tiledb` with the same URI, and any
 necessary arguments.
 
 
+Intermediate storage
+--------------------
+
+.. autosummary::
+   store
+
+In some cases, one may wish to store an intermediate result in long term
+storage. This differs from ``persist``, which is mainly used to manage
+intermediate results within Dask that don't necessarily have longevity.
+Also it differs from storing final results as these mark the end of the Dask
+graph. So lack anyway to be reused in other graphs without reloading the data.
+Intermediate storage is mainly useful in cases where the data is needed
+outside of Dask (e.g. on disk, in a database, in the cloud, etc.). It can
+be useful as a checkpoint for long running or flaky computations.
```

Comment:
```suggestion
be useful as a checkpoint for long running or computations that are prone to failure.
```

Flaky might not be well understood by non-native english speakers.

Reply:
Made a slightly different change, but hopefully that addresses this concern.